infrastructure/mesh-forwarder.tf

```diff
@@ -1,5 +1,5 @@
 locals {
-  is_mesh_forwarder_enable         = false
+  is_mesh_forwarder_enable         = local.is_force_destroy ? false : true
```

Comment:
Could we remove the dependency on is_force_destroy for this one?
is_mesh_forwarder_enabled should have it's own independent workspace checks.